### PR TITLE
Vegetarian Quirk

### PR DESCRIPTION
**Pull request by XDTM**
:cl: XDTM
add: Added the Vegetarian quirk, which costs 0 points and makes you dislike meat.
/:cl:

Because why not? Also, it could give a reason for the chef to prepare vegetarian dishes instead of the usual stuff.

### CODE REVIEW

**Comment by Time-Green** [comment]
Make vegans die when they eat meat

**Comment by GuyonBroadway** [comment]
Chef:"yeah there's a vegetarian option, you can fuck off!"

**Comment by MACIEKBAKI** [comment]
BREAKING NEWS!! A vegetarian lizard starves on Space Station 13 after the chef refused to make vegan food!
I like the potential of this

**Comment by MACIEKBAKI** [comment]
Wait. You know that quirk that makes you dislike any food you like and dislike you any food you like?
What if you took that and the vegan quirk?

**Comment by XDTM** [comment]
can't, i blacklisted the combination

**Comment by CosmicScientist** [comment]
Isn't egg defined as meat?

**Comment by Anonmare** [comment]
It is

**Comment by nicbn** [comment]
Egg needs it's own category

**Comment by improvedname** [comment]
Make it -1 point since most food is meat

**Comment by Farquaar** [comment]
> Make it -1 point since most food is meat

We have vending machines, two rooms and an entire job dedicated to making non-meat food.

**Comment by GuyonBroadway** [comment]
Veggies can happily partake of my greytide cheapskate special, better known as deep fried glass shards and metal rods.

File: code/datums/traits/neutral.dm [hunk 1]
```diff
@@ -10,6 +10,28 @@
...
+
+/datum/quirk/vegetarian/add()
+	var/mob/living/carbon/human/H = quirk_holder
+	var/datum/species/species = H.dna.species
+	species.liked_food &= ~MEAT
+	species.disliked_food |= MEAT
+
+/datum/quirk/vegetarian/remove()
+	var/mob/living/carbon/human/H = quirk_holder
+	if(H)
```
Comment (Cyberboss) [inline code/datums/traits/neutral.dm]: In what situation would this fail?

**Comment by XDTM** [comment]
I think it was the PR with the most upvotes so far

**Pull request merged**